app/models/site_config.rb

```diff
@@ -30,6 +30,8 @@ class SiteConfig < RailsSettings::Base
   field :community_member_label, type: :string, default: "user"
   field :community_action, type: :string
   field :tagline, type: :string
+  field :community_copyright_start_year, type: :integer,
+                                         default: ApplicationConfig["COMMUNITY_COPYRIGHT_START_YEAR"] || 2020
```

Comment:
Same as above, maybe replace 2020 with `Date.today.year`?

Reply:
On second thought, the problem with this is always that if the users never update the value explicitly the default will be wrong eventually. If we hardcode 2020 and forget to update it on Jan 1st it will be in the past and if we use my approach it will automatically move forward.

Reply:
Using current year makes sense. This is just the starter value so it should indeed always be current.